script/test/all

```diff
@@ -39,15 +39,13 @@ for version in $DOCKER_VERSIONS; do
 
   trap "on_exit" EXIT
 
-  repo="dockerswarm/dind"
-
   docker run \
     -d \
     --name "$daemon_container" \
     --privileged \
     --volume="/var/lib/docker" \
-    "$repo:$version" \
-    dockerd -H tcp://0.0.0.0:2375 $DOCKER_DAEMON_ARGS \
+    "docker:${version}-dind" \
+    sh -c "apk add --no-cache git && dockerd -H tcp://0.0.0.0:2375 $DOCKER_DAEMON_ARGS" \
```

Comment:
This is a bit ugly; there are variants with `git` installed, but not for the `dind` variant. Looks like it's only needed for a single test 😞